subsys/bluetooth/controller/ll_sw/pdu.h

```diff
@@ -18,6 +18,8 @@
 #define PDU_AC_LL_HEADER_SIZE  (offsetof(struct pdu_adv, payload))
 /* Advertisement channel maximum PDU size */
 #define PDU_AC_SIZE_MAX        (PDU_AC_LL_HEADER_SIZE + PDU_AC_PAYLOAD_SIZE_MAX)
+/* Advertisement channel Access Address */
+#define PDU_AC_ACCESS_ADDR     0x8e89bed6
```

Comment:
Make the Advertisement Channel Access Address less magic-number-y

Reply:
Approved, you can send a separate PR, so that we can merge this independent change it sooner

Reply:
PR is #21653